src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -493,11 +493,21 @@ private void GenerateFunctions(IEnumerable<Type> directTypes)
         /// </summary>
         private void InitializeFunctionDescriptors(Collection<FunctionMetadata> functionMetadata)
         {
-            _descriptorProviders = new List<FunctionDescriptorProvider>()
+            var language = _settingsManager.Configuration.GetSection("workers:config:language").Value;
```

Comment:
nit: `workers:language` would really work better. `config` in redundant. Also, this is used in a few places, so it would be good to introduce a constant.